### DISCUSSION TITLE
Pin cell output for quick interactive window dashboards

### DISCUSSION THREAD

**Discussion by krokosik** [microsoft/vscode-jupyter#15984] [New Ideas]
Hello,

I am currently using the interactive window for most of my Python work at our research lab. We often test new setups and want to prototype quickly, so it is rare that we create full GUIs with solutions like Qt for Python etc.

On the other hand I often rely on `ipywidgets` for quick and simple UIs that seamlessly intergrate with the code editor and allow me to easily fine tune parameters or toggle functionality with buttons without needing to restart the kernel or utilise separate windows. I really love having everything in VSCode, but the problem is that when I execute different cells, my Jupyter widgets are scrolled away and I need to rerun the cell that created them. It would be very useful to be able to somehow make an output cell *sticky* or just pin it at the top of the interactive window, so that I can have my widget available at a glance at all times.

I couldn't find such a feature available in the current extension, but maybe it wouldn't be that difficult to implement? I'm willing to help but have not contributed to the codebase before and would like to know if such a feature would be considered by the team :)

**Comment by DonJayamanne** [comment]
This is the closest feature request I can find that would solve this issue https://github.com/microsoft/vscode-jupyter/issues/1120
